src/wallet/rpcwallet.cpp

```diff
@@ -3233,6 +3309,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "walletpassphrasechange",   &walletpassphrasechange,   {"oldpassphrase","newpassphrase"} },
     { "wallet",             "walletpassphrase",         &walletpassphrase,         {"passphrase","timeout"} },
     { "wallet",             "removeprunedfunds",        &removeprunedfunds,        {"txid"} },
+    { "wallet",             "rescanblockchain",         &rescanblockchain,         {"start_height", "stop_height"} },
```

Comment:
Very tiny nit, but every other place *skips* the space between words in argument list. I.e. `{"start_height","stop_height"}`.